### PR TITLE
added readme, license and contribute guidelines

### PR DESCRIPTION
**Pull request by thebootable**
added readme and contribute guidelines with basic content just to get started

### CODE REVIEW

File: CONTRIBUTE.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Contribute
```
Comment (jbrooksuk) [inline CONTRIBUTE.md]: I think this file should live within `.github/CONTRIBUTING.md`. This makes the contributing guide discoverable by GitHub.

For example, https://github.com/cachethq/cachet/blob/2.4/.github/CONTRIBUTING.md

File: CONTRIBUTE.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Introduction
+
+First, thank you for considering contributing to cachet! It's people like you that make the open source community such a great community! 😊
```
Comment (jbrooksuk) [inline CONTRIBUTE.md]:
```suggestion
First, thank you for considering contributing to Cachet! It's people like you that make the open source community such a great community! 😊
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<p align="center">
+    <picture>
+      <source media="(prefers-color-scheme: dark)" srcset="https://cachethq.io/assets/cachet-logo-dark.svg">
+      <img alt="Cachet Logo" src="https://cachethq.io/assets/cachet-logo-light.svg">
+    </picture>
+</p>
+
+## Introduction
+
+This package provides the documentation to Cachet Core and is deployed to https://docs.cachethq.io/.
```
Comment (jbrooksuk) [inline README.md]:
```suggestion
This repository contains the documentation for Cachet and is deployed to https://docs.cachethq.io/.
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+## Contributing to the documentation
+
+Please refer to our description on [CONTRIBUTE.md](CONTRIBUTE.md) for details on how to contribute to this project.
```
Comment (jbrooksuk) [inline README.md]: This link will need updating.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+## Security Vulnerabilities
+
+Please review our [security policy](https://github.com/cachethq/cachet/security/policy) on how to report security vulnerabilities.
+
+## License
+
+Cachet Core is open-sourced software licensed under the [MIT license](LICENSE.md).
```
Comment (jbrooksuk) [inline README.md]:
```suggestion
Cachet Docs is open-sourced software licensed under the [MIT license](LICENSE.md).
```

We'll also need the `LICENSE.md` file creating in this repository.

File: CONTRIBUTE.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+The bigger the pull request, the longer it will take to review and merge. Try to break down large pull requests in smaller chunks that are easier to review and merge.
+It is also always helpful to have some context for your pull request. What was the purpose? Why does it matter to you?
+
+## Security Vulnerabilities
+
+Please review our [security policy](https://github.com/cachethq/cachet/security/policy) on how to report security vulnerabilities.
+
+## Questions
+
+If you have any questions, create an [issue](issue) (protip: do a quick search first to see if someone else didn't ask the same question before!).
```
Comment (jbrooksuk) [inline CONTRIBUTE.md]:
```suggestion
If you have any questions, please create a new [issue](https://github.com/cachethq/docs/issues).
```

**Review by jbrooksuk — CHANGES_REQUESTED** [review]

File: CONTRIBUTE.md (hunk 1 above)
Comment (thebootable) [reply CONTRIBUTE.md]: whops, very true.

File: README.md (hunk 5 above)
Comment (thebootable) [reply README.md]: I copied the LICENSE.md from the core repo

**Pull request merged**